### PR TITLE
#185392590 : A user should be able to view products per category

### PR DESCRIPTION
**Pull request by bergad37**
## What does this PR do?

Viewing products per category on landing page

## Description of Task to be completed?

From the list of categories, user can get items from a specific category

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

[#185392590](https://www.pivotaltracker.com/story/show/185392590)

##Screeshot

https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/104447243/2f58adbf-ebbd-4adc-b259-f597ee1f002d

Screen Shot 2023-06-19 at 20 58 06

### CODE REVIEW

**Review by mken1205 — APPROVED** [review]

File: src/containers/HomeBrowseCategory.jsx
Comment (irakevine) [inline src/containers/HomeBrowseCategory.jsx:1]: @bergad37 you can remove lines from 64 to 66 it's unnecessary comments

**Review by frankflej — APPROVED** [review]
Nice work!

File: src/containers/HomeBrowseCategory.jsx
Comment (bergad37) [reply src/containers/HomeBrowseCategory.jsx:1]: Done!

**Review by nishimweprince — APPROVED** [review]
Excellent work @bergad37 !!

**Review by irakevine — APPROVED** [review]
Great Work

**Review by Dyves1 — APPROVED** [review]

**Review by jcharles921 — APPROVED** [review]
LGTM !!!

**Pull request merged**